src/sentry/features/manager.py

```diff
@@ -150,6 +150,7 @@ def add(
         name: str,
         cls: type[Feature] = Feature,
         entity_feature_strategy: bool | FeatureHandlerStrategy = False,
+        default: bool | None = None,
```

Comment:
This should probably be required?

Reply:
Agreed, I was playing it safe with this change. Combined with your other suggestion I think making this required and having the default = False is a good solution.